module.dashboard.nf

```diff
@@ -0,0 +1,28 @@
+
+process demux_dash {
+    publishDir path: "${params.output_dir}/", pattern: "demux_dash", mode: 'copy'
```

Comment:
It's best to also include `overwrite: true` for `publishDir` in case a workflow is run repeatedly (in local testing mode)